typescript/cli/examples/fork/anvil-chains.yaml

```diff
@@ -13,6 +13,12 @@ anvil:
     name: Ether
     symbol: ETH
     decimals: 18
+alfajores:
```

Comment:
I want to use alfajores here since we need an account with a balance and 0xfaD1C94469700833717Fa8a3017278BC1cA8031C has plenty. Using an address funded on ethereum mainnet doesn't seem like a great choice. It's forked but the potential to move or use assets on ethL1 is high